Tests/Utilities/TestUniq.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2023, Rodrigo Tobar <rtobarc@gmail.com>.
  * Copyright (c) 2024, Daniel Gaston <tfd@tuta.io>.
+ *
```

Comment:
This change should be added to the commit where this file was first added.

Reply:
Fixed this and the above items, thanks for the feedback!